doc/portfile.7

```diff
@@ -293,37 +328,40 @@ can be omitted if there is only one distfile.
 .br
 .Sy Example:
 .Bd -literal -offset indent -compact
-checksums dictd-1.7.1.tar.gz md5 81317b86ea0a5df0163900ad2e6bb12c \e\ 
-	magic-words-1.7.1.tar.gz md5 897a005182928613eadd30c267ce9c5b
+checksums dictd-1.7.1.tar.gz md5 81317b86ea0a5df0163900ad2e6bb12c \e
+          magic-words-1.7.1.tar.gz md5 897a005182928613eadd30c267ce9c5b
 .Ed
 .br
 .Sy Example (ledit 1.11):
 .Bd -literal -offset indent -compact
-checksums md5 a2d38ba641682509c1e964ad699a9dd2 \e\ 
-	sha1 1fb6443b5fdf3c83787953f06282d256477c1288
+checksums md5 a2d38ba641682509c1e964ad699a9dd2 \e
+          sha1 1fb6443b5fdf3c83787953f06282d256477c1288
```

Comment:
Not in the scope of this PR, but this definitely needs to be corrected. We now want `rmd160`, `sha256`, `size`. By providing `sha1` and `md5` we give users exceptionally bad examples.